src/mixin.js

```diff
@@ -29,14 +29,14 @@ function assertIsDate(date, errMsg) {
 
 export default {
     statics: {
-        filterFormatOptions: function (obj) {
+        filterFormatOptions: function (obj, defaults) {
             return (this.formatOptions || []).reduce(function (opts, name) {
                 if (obj.hasOwnProperty(name)) {
                     opts[name] = obj[name];
                 }
 
                 return opts;
-            }, {});
+            }, defaults ? Object.create(defaults) : {});
```

Comment:
Prototype inheritance is safe to use here because [ECMA-402's `GetOption` abstract operation performs a `[[Get]]`](http://www.ecma-international.org/ecma-402/1.0/index.html#sec-9.2.9).